DiscordPy/bot.py

```diff
@@ -70,11 +72,12 @@ async def getHelp(ctx, *arg):
 
 @bot.command(name='meme')
 async def getMeme(ctx, *arg):
-    if (len(imageList) == 0):
-        hot_python = subreddit.new(limit=20)
+    delta_time = datetime.datetime.now() - prev_time
+    delta_seconds = delta_time.total_seconds()
+    if (len(imageList) == 0 or delta_seconds >= 60):

```

Comment:
Just double-checking,
You messaged me saying we wanted "60 calls per minute" to be the max, meaning we call the $meme function once per second.
Does this line 77 imply we only call it once per minute, rather than 60?